src/app/components/AllOrNone.tsx

```diff
@@ -24,6 +24,8 @@ export default styled(AllOrNone)`
   &,
   ${ButtonLink} {
     font-size: 1.4rem;
+    margin: 0;
+    overflow: hidden;
```

Comment:
Can this be `overflow: visible;`?
I've spotted this issue also on my pr: https://github.com/openstax/rex-web/pull/525 in the Edge browser and there only `overflow: visible` works

Reply:
![image](https://user-images.githubusercontent.com/25258810/76231847-6d77d480-61eb-11ea-92e3-499540c2132d.png)

this overflow is what brakes the buttons in safari, `overflow: hidden` fixes it

Reply:
I guess we can make it visible too